src/test/resources/net/sf/jabref/importer/fileformat/MedlinePlainImporterTestDOI.bib

```diff
@@ -1,3 +1,3 @@
 @TechReport{,
-  DOI                      = {10.1016/j.cpr.2005.02.002}
+  article-id                      = {doi:10.1016/j.cpr.2005.02.002}
```

Comment:
I wasn't sure if it is right if there stands ``doi:``. [Here] (https://www.nlm.nih.gov/bsd/mms/medlineelements.html#aid) it says, that doi has to come at the end in brackets. That's why I thought it would be wrong to add this as doi. Should I keep both possibilities ?